src/app/deleted/deleted.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit } from '@angular/core';
+import { UsersService } from '../users.service';
+
+@Component({
+  selector: 'app-deleted',
+  templateUrl: './deleted.component.html',
+  styleUrls: ['./deleted.component.css']
+})
+export class DeletedComponent {
+
+  deletedUsers = [];
+
+  constructor(private users: UsersService) {}
+
+  ngDoCheck(){
+    this.deletedUsers = this.users.getInActiveUsers();
```

Comment:
Any specific reason to load the users in `ngDoCheck` ?

Reply:
for reflecting immediate changes